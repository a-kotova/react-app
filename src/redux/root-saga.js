import { all, call } from 'redux-saga/effects';
import { fetchCollectionsStart } from './shop/shop.sagas';

export default function* rootSaga() {
  all([
    yield call(fetchCollectionsStart),
  ]);
}
