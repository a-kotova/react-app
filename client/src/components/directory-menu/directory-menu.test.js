import React from 'react';
import { shallow } from 'enzyme';
import DirectoryMenu from './directory-menu.component';
import { store } from '../../redux/store';

describe('Components: Directory Menu', () => {
  it('attempt', () => {
    const wrapper = shallow(<DirectoryMenu store={store} />).dive().dive();

    expect(wrapper).toMatchSnapshot();
  });
});
