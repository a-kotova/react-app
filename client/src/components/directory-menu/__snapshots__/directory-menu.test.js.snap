// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: Directory Menu attempt 1`] = `
<DirectoryMenuContainer>
  <withRouter(MenuItem)
    imageUrl="https://i.ibb.co/cvpntL1/hats.png"
    key="1"
    linkUrl="shop/hats"
    title="hats"
  />
  <withRouter(MenuItem)
    imageUrl="https://i.ibb.co/px2tCc3/jackets.png"
    key="2"
    linkUrl="shop/jackets"
    title="jackets"
  />
  <withRouter(MenuItem)
    imageUrl="https://i.ibb.co/0jqHpnp/sneakers.png"
    key="3"
    linkUrl="shop/sneakers"
    title="sneakers"
  />
  <withRouter(MenuItem)
    imageUrl="https://i.ibb.co/GCCdy8t/womens.png"
    key="4"
    linkUrl="shop/womens"
    size="large"
    title="womens"
  />
  <withRouter(MenuItem)
    imageUrl="https://i.ibb.co/R70vBrQ/men.png"
    key="5"
    linkUrl="shop/mens"
    size="large"
    title="mens"
  />
</DirectoryMenuContainer>
`;
